### DISCUSSION TITLE
Showcasing My GitHub Journey

### DISCUSSION THREAD

**Discussion by vivek20Eng** [community/community#129035] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Welcome to the conversation about my GitHub profile! Here, you can explore the repositories I've contributed to, the projects I'm passionate about, and the milestones I've achieved. Let's dive into discussions about my coding journey, share insights on open-source projects, and exchange ideas to improve our collaborative efforts. Feel free to ask questions, leave feedback, or just say hi!

**Comment by ebndev** [comment]
Hi @vivek20Eng, we’re glad you’re here 😀

Thanks for sharing your profile with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations.

However, please feel encouraged to share your profile and engage with other users in our [Showcase Your GitHub Profile! 🌟](https://github.com/orgs/community/discussions/121833) thread!
